exercises/practice/two-bucket/.meta/reference/bucket.bal

```diff
@@ -0,0 +1,38 @@
+class Bucket {
+    final string name;
+    final int size;
+    int amount;
```

Comment:
Can make these `private` since we don't access them directly. 

```suggestion
    private final string name;
    private final int size;
    private int amount;
```